assets/js/base/components/sort-select/index.js

```diff
@@ -4,7 +4,7 @@
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import Label from '@woocommerce/base-components/label';
-import { withInstanceId } from 'wordpress-compose';
+import { withInstanceId } from '@wordpress/compose';
```

Comment:
This component is used in _Reviews_ block, which are available in legacy environments, so we can't use the lastest version of `@wordpress/compose` because it relies on hooks which are not available in WP <= 5.2.

The change included in this PR makes it so we load the external instead of the latest version of the library.